<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="app.psc.testcases.userverification.userverificationtestcases" tests="8" time="169.399" ignored="1" failures="0" timestamp="29 Mar 2018 11:44:39 GMT" skipped="0" hostname="XWJPMDD035" errors="2">
  <testcase name="TC007_02_Verify_Login_Flow_Existing_User_New_Browser_Invalid_input" time="58.584" classname="app.psc.testcases.userverification.userverificationtestcases"/>
  <testcase name="TC007_02_Verify_Login_Flow_Existing_User_New_Browser_Invalid_input" time="5.807" classname="app.psc.testcases.userverification.userverificationtestcases"/>
  <testcase name="DriverQuite" time="0.000" classname="app.psc.testcases.userverification.userverificationtestcases">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at app.psc.testcases.userverification.userverificationtestcases.DriverQuite(userverificationtestcases.java:312)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:357)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- DriverQuite -->
  <testcase name="TC007_02_Verify_Login_Flow_Existing_User_New_Browser_Invalid_input" time="5.808" classname="app.psc.testcases.userverification.userverificationtestcases"/>
  <testcase name="TC007_02_Verify_Login_Flow_Existing_User_New_Browser_Invalid_input" time="6.179" classname="app.psc.testcases.userverification.userverificationtestcases"/>
  <testcase name="TC007_03_Verify_Login_Flow_Existing_User_withing_same_browser_session" time="0.807" classname="app.psc.testcases.userverification.userverificationtestcases">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at app.psc.testcases.userverification.userverificationtestcases.TC007_03_Verify_Login_Flow_Existing_User_withing_same_browser_session(userverificationtestcases.java:262)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- TC007_03_Verify_Login_Flow_Existing_User_withing_same_browser_session -->
  <testcase name="TC007_02_Verify_Login_Flow_Existing_User_New_Browser_Invalid_input" time="5.810" classname="app.psc.testcases.userverification.userverificationtestcases"/>
  <testcase name="TC007_01_Verify_Login_Flow_Existing_User_NewBrowser_Valid_Input" time="86.404" classname="app.psc.testcases.userverification.userverificationtestcases"/>
  <testcase name="Verify_TUDS_table_for_Magic_login" classname="app.psc.testcases.userverification.userverificationtestcases">
    <skipped/>
  </testcase> <!-- Verify_TUDS_table_for_Magic_login -->
</testsuite> <!-- app.psc.testcases.userverification.userverificationtestcases -->
